ci/check-params-env.sh

```diff
@@ -407,66 +460,75 @@ function check_image() {
-    [[ "${LINE}" = *[[:space:]]* ]] && {
-        echo "ERROR: Line contains white-space and it shouldn't!"
-        echo "--------------------------------------------------"
-        ret_code=1
-        continue
-    }
-    [[ "${LINE}" != *=* ]] && {
-        echo "ERROR: Line doesn't contain '=' and it should!"
-        echo "----------------------------------------------"
-        ret_code=1
-        continue
-    }
+process_file() {
+    while IFS= read -r LINE; do
+        echo "Checking format of: '${LINE}'"
+        [[ "${LINE}" = *[[:space:]]* ]] && {
+            echo "ERROR: Line contains white-space and it shouldn't!"
+            echo "--------------------------------------------------"
+            ret_code=1
+            continue
+        }
+        [[ "${LINE}" != *=* ]] && {
+            echo "ERROR: Line doesn't contain '=' and it should!"
+            echo "----------------------------------------------"
+            ret_code=1
+            continue
+        }
 
-    IMAGE_VARIABLE=$(echo "${LINE}" | cut --delimiter '=' --field 1)
-    IMAGE_URL=$(echo "${LINE}" | cut --delimiter '=' --field 2)
+        IMAGE_VARIABLE=$(echo "${LINE}" | cut --delimiter '=' --field 1)
+        IMAGE_URL=$(echo "${LINE}" | cut --delimiter '=' --field 2)
 
-    test -n "${IMAGE_VARIABLE}" || {
-        echo "ERROR: Couldn't parse image variable - got empty value!"
-        echo "-------------------------------------------------------"
-        ret_code=1
-        continue
-    }
+        test -n "${IMAGE_VARIABLE}" || {
+            echo "ERROR: Couldn't parse image variable - got empty value!"
+            echo "-------------------------------------------------------"
+            ret_code=1
+            continue
+        }
 
-    test -n "${IMAGE_URL}" || {
-        echo "ERROR: Couldn't parse image URL - got empty value!"
-        echo "--------------------------------------------------"
-        ret_code=1
-        continue
-    }
+        test -n "${IMAGE_URL}" || {
+            echo "ERROR: Couldn't parse image URL - got empty value!"
+            echo "--------------------------------------------------"
+            ret_code=1
+            continue
+        }
 
-    check_image "${IMAGE_VARIABLE}" "${IMAGE_URL}" || {
-        echo "ERROR: Image definition for '${IMAGE_VARIABLE}' isn't okay!"
-        echo "------------------------"
-        ret_code=1
-        continue
-    }
-done < "${PARAMS_ENV_PATH}"
+        check_image "${IMAGE_VARIABLE}" "${IMAGE_URL}" || {
+            echo "ERROR: Image definition for '${IMAGE_VARIABLE}' isn't okay!"
+            echo "------------------------"
+            ret_code=1
+            continue
+        }
+    done < "${1}"
+}
+
+# process_file "${PARAMS_ENV_PATH}" || {ret_code=1}
+# if test "${ret_code}" -eq 0; then
+#     echo "Validation of '${PARAMS_ENV_PATH}' was successful! Congrats :)"
+# else
+#     echo "The '${PARAMS_ENV_PATH}' file isn't valid, please check above!"
+# fi
```

Comment:
Is this commented out on purpose?

Maybe we should put this into a `for` cycle iteration over these two files to avoid code duplicity?

Reply:
Yes, made the change.